### DISCUSSION TITLE
Module Management is slow

### DISCUSSION THREAD

**Discussion by Sharafudeen** [oqtane/oqtane.framework#3311] [General] [answered]
Hi,

How can I improve loading performance for all admin modules in Oqtane? Especially Module Management module..

**Comment by sbwalker** [comment, marked as answer]
Module Management calls an external service to determine if updates exist to any of the installed modules. You can disable this in System Info and Options tab by setting the Package Manager to blank. This also means you will not be able to install Modules, Themes, or Translations from the Oqtane Marketplace or perform an automated System Update.
